Location.md

```diff
@@ -20,6 +20,32 @@ See HTML.
 
 See HTML.
 
+## Location.toString ( )
+
+1. Let _hrefDesc_ be DefaultInternalMethod([[GetOwnProperty]\], this, "href").
+
+1. Return the result of calling _hrefDesc_.[[Get]\].
+
+The property has the attributes { [[Writable]]: false, [[Enumerable]]: true, [[Configurable]]: false }.
+
+## Location.toJSON
+
+The initial value of Location.toJSON is undefined.
+
+The property has the attributes { [[Writable]]: false, [[Enumerable]]: true, [[Configurable]]: false }.
+
+## Location.valueOf ( )
+
+1. Return this.
```

Comment:
Why should valueOf return a string or a number?  That's not what http://www.ecma-international.org/ecma-262/6.0/#sec-object.prototype.valueof does, for example.